common/src/domain/pokemon/skill-pokemon.ts

```diff
@@ -720,6 +760,7 @@ export const BONSLY: Pokemon = {
 
 export const DRIFLOON: Pokemon = {
```

Comment:
Oops, looks like this has been out-of-order for a while! Can you add another commit putting Drifloon and Drifblim between Wynaut and Bonsly?